CONTRIBUTING

```diff
@@ -0,0 +1,35 @@
+# Contributing
+
+Hey, thanks for your interest in contributing to the NYU CS Wiki! Please review this document **before making a pull request**.
+
+## Guidelines 
+
+- All articles are written in a format known as [Markdown](https://commonmark.org/help/). Don't be worried! It's very simple to write in once you get the hang of it; for reference, you can look at any of the pages in the /docs folder. 
+- Looking in /docs, there are 3 main categories: Academics/Opportunities/Recreation. Let's take a look at /docs/Academics for reference. You can see an ``academics.md`` file which serves as an overview (visible when clicking on a category). From here, there is ``classes.md``, ``schedule.md``, etc which talk about their respective subjects. However, if there's a topic that requires multiple articles, you can create a folder (ex. /Academics/Tandon) and create an overview file and more pages from here. 
```

Comment:
Might be good to add links to the files here.

You use "/" to separate the categories, I would recommend using "," (otherwise it can look like a file path).